kinto_client/tests/test_cli_utils.py

```diff
@@ -38,6 +38,20 @@ def test_set_parser_server_options_adds_arguments_on_existing_parser(self):
                                    *ALL_PARAMETERS)
         assert len(parser._actions) == 9
 
+    def test_set_parser_server_options_can_ignore_bucket_and_collection(self):
+        parser = cli_utils.set_parser_server_options(
+            with_bucket=None, with_collection=None)
+        parameters = [
+            ['-h', '--help'],
+            ['-s', '--server'],
+            ['-a', '--auth'],
+            ['-v', '--verbose'],
+            ['-q', '--quiet'],
+            ['-D', '--debug'],
+        ]
+        self.assert_option_strings(parser, *parameters)
+        assert len(parser._actions) == 6
```

Comment:
This can be painful to maintain. But I can't suggest anything better (except grep in `--help`)